Makefile

```diff
@@ -152,7 +152,7 @@ all-containers:
 	$(MAKE) container BIN=velero-restic-restore-helper
 	$(MAKE) build-fsfreeze
 
-container: verify test .container-$(DOTFILE_IMAGE) container-name
+container: .container-$(DOTFILE_IMAGE) container-name
```

Comment:
Just to confirm - the removal of verify/test here is because we've already run them as part of the "build", so we don't need to re-run them?

Reply:
That's correct.